fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -289,13 +289,17 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 
 	GetRangeLimits randomLimits() {
 		// TODO : fix knobs for row_unlimited
-		int rowLimits = deterministicRandom()->randomInt(1, keysCount.getValue() + 1);
+		int rowLimits = deterministicRandom()->randomInt(0, keysCount.getValue() + 1);
 		// The largest key's bytes is longest prefix bytes + 1(for '/') + generated key bytes
 		// 8 here refers to bytes of KeyValueRef
 		int byteLimits = deterministicRandom()->randomInt(
 		    1, keysCount.getValue() * (keyBytes + (rangeCount + 1) + valBytes + 8) + 1);
 
-		return GetRangeLimits(rowLimits, byteLimits);
+		auto limit = GetRangeLimits(rowLimits, byteLimits);
+		// minRows is always initilized to 1
+		if (limit.rows == 0)
+			limit.minRows = 0;
```

Comment:
Probably add a constructor `GetRangeLimits(rowLimits, minRows, byteLimits)` or in the constructor initialize minRows(min(rowLimits, 1)?

Reply:
I feel the GetRangeLimits has `minRows` initialized to 1 for a long time.
This is a special use case and in practice, the `minRows` is decrementing by calling `GetRangeLimits::decrement`

As it's a single-use case, I feel it's better to use it like this rather than changing the old constructor.